internal/rbd/rbd_util.go

```diff
@@ -816,20 +816,20 @@ func getSnapshotMetadata(ctx context.Context, pSnapOpts *rbdSnapshot, cr *util.C
 	return nil
 }
 
-// imageInfo strongly typed JSON spec for image info
-type imageInfo struct {
+// ImageInfo strongly typed JSON spec for image info
+type ImageInfo struct {
 	ObjectUUID string   `json:"name"`
 	Size       int64    `json:"size"`
 	Features   []string `json:"features"`
 	CreatedAt  string   `json:"create_timestamp"`
 }
 
-// getImageInfo queries rbd about the given image and returns its metadata, and returns
+// GetImageInfo queries rbd about the given image and returns its metadata, and returns
 // ErrImageNotFound if provided image is not found
-func (rv *rbdVolume) getImageInfo(ctx context.Context, monitors string, cr *util.Credentials) (imageInfo, error) {
+func GetImageInfo(ctx context.Context, monitors string, cr *util.Credentials, poolName, imageName string) (ImageInfo, error) {
```

Comment:
keep it as the Method only. and don't change the function signature.

Reply:
But rbdVolume is an Internal variable , should I change it to RbdVolume ?